Meta/build-image-qemu.sh

```diff
@@ -55,6 +55,10 @@ elif [ "$(uname -s)" = "SerenityOS" ]; then
 else
     E2FSCK="/usr/sbin/e2fsck"
 
+    if [ ! -f "$E2FSCK" ]; then
+        E2FSCK=$(command -v e2fsck)
```

Comment:
We should do this first, and then fallback to manually attempting to search? 
That way if you have a system version, and a version you explicitly added to your path, the version in your path is respected, as I think most people would hope and assume it would be?

Reply:
Just as a note, that would also remove the special case for SerenityOS. In fact, the only reason why this is done manually at all is because _some_ unnamed distributions refuse to add `sbin` directories to their `PATH`.

Reply:
@bgianfo @timschumi Yeah, that makes sense, I have changed my commit to do that. LMK if its better now:)